Automattic/wp-calypso
# Jetpack Pricing: Add new realtime products to the Jetpack pricing page

#### Changes proposed in this Pull Request

This PR removes the daily products from the pricing page, and instead only displays the new real-time products.

Note that I have added placeholder prices until real products are added to the Store.

<img width="1092" alt="image" src="https://user-images.githubusercontent.com/42627630/129763696-814440ca-0107-42be-99b7-d9746b655c10.png">

<img width="1101" alt="image" src="https://user-images.githubusercontent.com/42627630/129763743-1eada3c4-6665-4bbe-adaa-b1578f903ba4.png">

<img width="1128" alt="image" src="https://user-images.githubusercontent.com/42627630/129763777-ff9a4d45-89e0-4e49-9883-f8a4e82440d2.png">

#### Testing instructions



1. Apply patch D65408 to your sandbox, enable global HTTP traffic, and point wordpress.com and public-api.wordpress.com to your sandbox.
2. Visit `http://jetpack.cloud.localhost:3001/pricing?flags=jetpack/only-realtime-products` and verify that you see the new products displaying and that the prices are correct.
2. Visit `http://jetpack.cloud.localhost:3001/pricing` and verify that you do not see the new products displaying.

Test the purchase flow:
1. Start both Jetpack Cloud and Calypso: `yarn start` and `yarn start-jetpack-cloud-p`.
1. Follow step #1 from above.
2. Create a new Jurassic Ninja site and connect it.
3. When you come to the pricing page, change `cloud.jetpack.com` to `jetpack.cloud.localhost:3001` and append `&flags=jetpack/only-realtime-products` to the end of the URL. Verify that you see the new products.
4. Copy one of the links to purchase a new product, and change `https://wordpress.com/` to `calypso.localhost:3000/`.
5. Verify that you can purchase the new product.



Related to 1200412004370260-as-1200532973279630
